zocalo/service/dispatcher.py

```diff
@@ -0,0 +1,250 @@
+from __future__ import absolute_import, division, print_function
+
+import copy
+import errno
+import json
+import os
+import re
+import time
+import timeit
+import uuid
+
+import workflows.recipe
+from workflows.services.common_service import CommonService
+
+
+class Dispatcher(CommonService):
+    """Single point of contact service that takes in job meta-information
+     (say, a data collection ID), a processing recipe, a list of recipes,
+     or pointers to recipes stored elsewhere, and mangles these into something
+     that can be processed by downstream services.
+  """
+
+    # Human readable service name
+    _service_name = "Dispatcher"
+
+    # Logger name
+    _logger_name = "services.dispatcher"
+
+    # Define a base path where your recipes are located (accessed with zocalo.go -r $recipename)
+    _recipe_basepath = None
+
+    # Define a logbook if you would like to do logging
+    _logbook = None
```

Comment:
I think everything lookbook related should be left out. Replace with _before-filter_ and (_after-filter_ or _before-dispatch_) hook.

Reply:
Should these hooks take arguments?